docs/source/query-plans.mdx

```diff
@@ -444,6 +444,14 @@ You can view the query plan for a particular operation in any of the following w
     * Note that you must [register your federated graph](./managed-federation/setup/) with GraphOS to view query plans in the Explorer.
 * As direct output from the `@apollo/gateway` library (see below)
 
+### Outputting query plans with headers
+In the latest versions of Apollo Router and Gateway you can pass the following headers to return the query plans in the response extensions:
```

Comment:
Could you please specify the versions to keep this more evergreen?

Reply:
Updated!